west.yml

```diff
@@ -34,7 +34,7 @@ manifest:
 
     - name: golioth-zephyr-boards
       path: deps/modules/lib/golioth-boards
-      revision: v2.0.0
+      revision: szczys/fix-elixir-sysbuild
```

Comment:
### Before merging:

Update this to use tag of a new release cut after https://github.com/golioth/golioth-zephyr-boards/pull/23 merges.